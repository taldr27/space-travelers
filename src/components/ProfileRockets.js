import PropTypes from 'prop-types';

const ProfileRockets = ({ name }) => (
  <>
    <tr>
      <td>{name}</td>
    </tr>
  </>
);

export default ProfileRockets;

ProfileRockets.propTypes = {
  name: PropTypes.string.isRequired,
};
