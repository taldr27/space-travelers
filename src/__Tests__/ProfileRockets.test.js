import React from 'react';
import { render } from '@testing-library/react';
import ProfileRockets from '../components/ProfileRockets';

describe('Profile Rockets Testing', () => {
  it('Should render Profile Rockets table and compare with Snapshot', () => {
    const { asFragment } = render(<ProfileRockets name="Thaicom" />);
    expect(asFragment(<ProfileRockets name="Thaicom" />)).toMatchSnapshot();
  });
});
