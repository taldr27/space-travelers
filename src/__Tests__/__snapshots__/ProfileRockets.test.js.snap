// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Rockets Testing Should render Profile Rockets table and compare with Snapshot 1`] = `
<DocumentFragment>
  Thaicom
</DocumentFragment>
`;
